production/bin_aws/start.sh

```diff
@@ -55,14 +69,14 @@ if [ -n "${PRIMARY}" ]; then
 fi
 
 # compile assets in the background, particularly the admin interface
-su ${DEPLOYUSER} -c "nice bundle exec rake assets:precompile" &
+#su ${DEPLOYUSER} -c "nice bundle exec rake assets:precompile" &
```

Comment:
No need to comment out lines, better to remove them completely (along with their comments). Git history remains.